src/main/webapp/app/admin/logs/logs.vue

```diff
@@ -1,6 +1,8 @@
 <template>
-    <div class="table-responsive">
-        <h2 id="logs-page-heading">Logs</h2>
+    <div>
```

Comment:
changes not related to this PR. If needed, it should be done in a separate PR and maybe to Vue.js blueprint too

Reply:
Yes, we need to seperate new loggers component for instances to get less conflict when we upgrade the app. In my next commit I will reset this changes.